src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/shim.h

```diff
@@ -263,7 +263,7 @@ using addr_type = uint64_t;
 
       std::string getSimulatorType(const std::string& binaryDirectory);
       void createPreSimScript(const std::string& wcfgFilePath, std::string& preSimScriptPath);
-      void constructQueryTable();
+      void constructQueryTable();     
```

Comment:
Do not add useless space.
Please check your code before committing and configure your text-editor or IDE.